mobile/src/main/java/ch/epfl/sdp/mobile/state/StatefulVisitedProfileScreen.kt

```diff
@@ -30,17 +29,15 @@ import kotlinx.coroutines.launch
 class FetchedUserProfileScreenState(
     private val currentUser: AuthenticatedUser,
     private val user: Profile,
-    actions: State<ProfileActions>,
-    onChallengeClickAction: State<(String) -> Unit>,
```

Comment:
Missing / Not up to date documentation.

Reply:
Updated in [02a8f6a](https://github.com/epfl-SDP/android/pull/303/commits/02a8f6adccec26f453c8dbc6542da6ab4097754b)